src/Json.Schema.Validation/RuleResources.resx

```diff
@@ -200,117 +200,117 @@
+    <value>An array has more elements than the number of elements in the array specified by the schema's 'items' property, and the schema does not permit additional array items.</value>
   </data>
   <data name="RuleDescriptionTooFewProperties" xml:space="preserve">
-    <value>An object has fewer properties than permitted by the schema's "minProperties" property.</value>
+    <value>An object has fewer properties than permitted by the schema's 'minProperties' property.</value>
   </data>
   <data name="RuleDescriptionTooManyArrayItems" xml:space="preserve">
-    <value>An array has more elements than permitted by the schema's "maxItems" property.</value>
+    <value>An array has more elements than permitted by the schema's 'maxItems' property.</value>
   </data>
   <data name="RuleDescriptionTooManyProperties" xml:space="preserve">
-    <value>An object has more properties than permitted by the schema's "maxProperties" property.</value>
+    <value>An object has more properties than permitted by the schema's 'maxProperties' property.</value>
   </data>
   <data name="RuleDescriptionValidatesAgainstNotSchema" xml:space="preserve">
-    <value>An instance successfully validates against the schema specified by the schema's "not" property.</value>
+    <value>An instance successfully validates against the schema specified by the schema's 'not' property.</value>
   </data>
   <data name="RuleDescriptionValueTooLarge" xml:space="preserve">
-    <value>A numeric value is greater than the maximum value permitted by the schema's "maximum" property.</value>
+    <value>A numeric value is greater than the maximum value permitted by the schema's 'maximum' property.</value>
   </data>
   <data name="RuleDescriptionValueTooLargeExclusive" xml:space="preserve">
-    <value>A numeric value is greater than or equal to the exclusive maximum value permitted by the schema's "maximum" and "exclusiveMaximum" properties.</value>
+    <value>A numeric value is greater than or equal to the exclusive maximum value permitted by the schema's 'maximum' and 'exclusiveMaximum' properties.</value>
   </data>
   <data name="RuleDescriptionValueTooSmall" xml:space="preserve">
-    <value>A numeric value is less than the minimum value permitted by the schema's "minimum" property.</value>
+    <value>A numeric value is less than the minimum value permitted by the schema's 'minimum' property.</value>
   </data>
   <data name="RuleDescriptionValueTooSmallExclusive" xml:space="preserve">
-    <value>A numeric value is less than or equal to the exclusive minimum value permitted by the schema's "minimum" and "exclusiveMinimum" properties.</value>
+    <value>A numeric value is less than or equal to the exclusive minimum value permitted by the schema's 'minimum' and 'exclusiveMinimum' properties.</value>
   </data>
```

Comment:
what's the value? I'll stop making this observation but many of these rules don't provide per-instance information on the bad value that was observed/

Reply:
This is a rule description, not a result message. The corresponding result message does mention the values.